internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -17,15 +17,271 @@ Parameters:
+    Type: AWS::IAM::Role
+    Properties:
+      {{- if .PermissionsBoundary}}
+      PermissionsBoundary: !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:policy/{{.PermissionsBoundary}}
+      {{- end}}
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - states.amazonaws.com
+            Action: sts:AssumeRole
+      Policies:
+        - PolicyName: InvokeLambda
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action: lambda:InvokeFunction
+                Resource: !GetAtt CopyAssetsFunction.Arn
+        - PolicyName: S3Access
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action: s3:GetObject
+                Resource: arn:aws:s3:::{{ .AssetMappingFileBucket }}/{{ .AssetMappingFilePath }}
+
+  CopyAssetsFunction:
+    Metadata:
+      aws:copilot:description: A lambda function to copy assets into the static site bucket
+    Type: AWS::Lambda::Function
+    Properties:
+      Handler: index.handler
+      Role: !GetAtt CopyAssetsFunctionRole.Arn
+      Runtime: nodejs16.x
+      Timeout: 60
+      Code:
+        ZipFile: |
+          var aws = require("aws-sdk");
+
+          exports.handler = (event, context, callback) => {
+            const s3 = new aws.S3();
+            console.log("copy from cache bucket event:", event)
+            const params = {
+              CopySource: event.srcBucket + "/" + event.mapping.path,
+              Bucket: event.destBucket,
+              Key: event.mapping.destPath
+            };
+            console.log("copy object params", params)
+
+            s3.copyObject(params, (err, data) => {
+              if (err) {
+                console.log("error copying object", err)
+                callback(err);
+                return;
+              }
+
+              console.log("copied object: ", data)
+              callback(null, "Success");
+            });
+          };
+
+  CopyAssetsFunctionRole:
+    Metadata:
+      aws:copilot:description: An IAM Role for the lambda that copies assets into the static site bucket
+    Type: AWS::IAM::Role
+    Properties:
+      {{- if .PermissionsBoundary}}
+      PermissionsBoundary: !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:policy/{{.PermissionsBoundary}}
+      {{- end}}
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+            Action: sts:AssumeRole
+      Policies:
+        - PolicyName: CopyFromS3
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action:
+                  - s3:GetObject
+                  - s3:GetObjectTagging
```

Comment:
Just to double check: is this required for getting an object?

Reply:
`s3:GetObject` _defiantly_ is. `s3:GetObjectTagging` is only required if the object has tags, which....i don't think we'll have tags on our objects? so maybe it's not needed? let me test without it!

Reply:
ok i tested and it looks like we don't need it since we don't add any tags on our uploads! i'll remove it 👍